src/router/index.ts

```diff
@@ -10,9 +10,20 @@ export const routes: Array<RouteConfig> = [
   ...treesRoutes,
 ]
 
-const router = new VueRouter({
+const scrollBehavior = (to) => {
+  if (to.params.field) {
+    const anchor = `${to.params.blockId}.${to.params.field}`
+    const ele = document.getElementById(anchor)
+    if (ele) {
+      ele.scrollIntoView({ behavior: 'smooth', block: 'center' })
+    }
```

Comment:
can we add an `else` and log a debug here? it would be interesting to see something if the element was not found
eg: `console.debug('Dee linking: cannot scroll to ', anchor, ' as the DOM element is not found')`

Reply:
Done